doc/guides/build/kconfig/tips.rst

```diff
@@ -6,13 +6,6 @@ Kconfig - Tips and Best Practices
 This page covers some Kconfig best practices and explains some Kconfig
 behaviors and features that might be cryptic or that are easily overlooked.
 
-.. note::
-
-   The official Kconfig documentation is `kconfig-language.rst
-   <https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html>`__
-   and `kconfig-macro-language.rst
-   <https://www.kernel.org/doc/html/latest/kbuild/kconfig-macro-language.html>`__.
-
```

Comment:
I don't think we should remove the links to the official Kconfig documentation from the "Kconfig - Tips and Best Practices" page because it is, in fact, a very useful tip.

Also the `Bluetooth:` prefix seems to be irrelevant.

Reply:
> Not sure this small note needs a page of its own, but others may disagree :)
> What does the doc change have to do with Bluetooth?

1) Yes, the page is small.  The point is to get the pointer to the official docs into the Kconfig doc table of contents, so that they are more visible.  A separate page seems to be the way to do this.
If people do not think this is a good change, I am fine with that.  But it would have helped me.

2) Nothing - will change

Reply:
1) They are not removed, only moved.  I considered keeping them in the original location as well, but decided against it, as that would duplicate information.  (A certain way to forget to update one of the instances if the links ever change.)
I could add a reference to the new page instead - would that be better?

2) Yes, it is.  It was added by force of habit - it is what I usually write first in a PR subject line.  Will fix.